### ISSUE TITLE
SLAUGHTER

### ISSUE THREAD

**Issue by w1n5t0ngr** [deltaxflux/fluxion#231]
### Which version of fluxion are you using?
FLUXION 0.23 rev. 79

### What distribution of Linux (including the version) are you running it on?
4.6.0-kali1-amd64

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Turbo-X WiFi USB Adapter N300 WLN2-300 Dual Antenna
(RALINK RT2870)

### Which is the driver for it?
rt2800usb

### Does it support injection ( output of aireplay-ng -9)
18:05:25  Trying broadcast probe requests...
18:05:25  Injection is working!
18:05:26  Found 2 APs
...

### General description of your issue along with the steps to reproduce it
After finding and choosing the wifi that I like I select option 3 for WPS-SLAUGHTER.
In attack menu I select option 4...
Wash gives me the "bad FCS error" (please check the screenshot below).

I suggest giving to the user the choice in which the program will let him choose if he wants to ignore bad packages or not.
Basically you need to add the **-C or --ignore-fcs** parameter to wherever you execute the wash -i command.

Hope it helps. 

Keep it up 👍 

![capture](https://cloud.githubusercontent.com/assets/4209889/19811747/624c1314-9d3b-11e6-95d8-8d4c2e12a7ea.PNG)

**Comment by deltaxflux** [comment]
Its a bug I will work on it

**Comment by w1n5t0ngr** [comment]
Glad I could help.
Please review the following lines (fluxion file) :
2141
2286
2442
2600
2763

**Comment by deltaxflux** [comment]
Thanks for reporting
